src/extensions/extensionsHelper.ts

```diff
@@ -824,7 +1037,7 @@ export async function uploadExtensionVersionFromLocalSource(args: {
     throw err;
   }
   await deleteUploadedSource(objectPath);
-  return res;
+  return;
```

Comment:
Need to return res here. We need to return the extensionVersion from ext:dev:upload so that it supports the --json flag.